libs/gltfio/src/AssetLoader.cpp

```diff
@@ -531,8 +537,8 @@ void FAssetLoader::createRenderable(const cgltf_node* node, Entity entity, const
     }
 }
 
-bool FAssetLoader::createPrimitive(const cgltf_primitive* inPrim, Primitive* outPrim,
-        const UvMap& uvmap, const char* name) {
+bool FAssetLoader::createPrimitive(const cgltf_data* srcAsset, const cgltf_primitive* inPrim,
```

Comment:
Good question. No specific reason, apart from keeping signatures of similar methods similar. And some groundwork for using extras in mentioned methods as well.

From a purist perspective, these changes are not meaningful within the scope of this exact PR alone

Reply:
Yes, please don't make changes that are not necessary for the scope of this PR -- this is because it'll make things more confusing in the future, when we go back to this change. Thanks!

Reply:
Updated